common/src/main/java/org/dash/wallet/common/services/SystemActionsService.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Dash Core Group.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.dash.wallet.common.services
+
+interface SystemActionsService {
+    fun copyText(text: String, label: String, showToast: Boolean = true)
+    fun shareText(text: String)
+    fun reviewApp()
+    fun openAppSettings()
+    fun openNotificationChannelSettings(channel: String)
+    fun openNotificationSettings()
+}
```

Comment:
Such an interface could be useful in the case we ever use multiplatform: both android and ios can call on it from shared code, while implementations will be specific to the platform.